dle/static/search/es_search.js

```diff
@@ -0,0 +1,101 @@
+    /* global instantsearch algoliasearch */
```

Comment:
This is adding a second es_search.js file - wanted to make sure that was intentional.

Reply:
Good catch, it was not ... Should probably add `/dle/static` to `.gitignore`